packages/neon-core/src/wallet/components/AssetBalance.ts

```diff
@@ -29,6 +34,11 @@ export class AssetBalance implements NeonObject {
     this.unconfirmed = abLike.unconfirmed
       ? abLike.unconfirmed.map((coin) => new Coin(coin))
       : [];
+
+    if (!abLike.hash || !isAssetHash(abLike.hash)) {
+      throw new Error(EXCEPTION_HASH_MSG);

```

Comment:
Please inline the message. We do not need to export error strings.

Reply:
ok, I will do it